include/ssm-internal.h

```diff
@@ -123,6 +123,9 @@ void ssm_tick(void);
  */
 #define ssm_adt_size(vc) (sizeof(struct ssm_adt1) + sizeof(ssm_value_t) * ((vc) - 1))
 
+
+#define ssm_closure_size(vc) (sizeof(struct ssm_closure1) + (vc ? sizeof(ssm_value_t) * ((vc) - 1) : -sizeof(ssm_value_t)))
```

Comment:
Do you need the ternary? I may be wrong but I thought C casts arithmetic to (`signed`?) `int` before converting back.

Reply:
This was left over from the original implementation of this function.